<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasketViewItem</class>
 <widget class="QWidget" name="BasketViewItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="item_picture">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>50</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="item_name">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>10</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="number">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>260</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>تعداد:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="item_number">
   <property name="geometry">
    <rect>
     <x>613</x>
     <y>260</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="remove">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>حذف</string>
   </property>
  </widget>
  <widget class="QLabel" name="color">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>60</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="total_price">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>255</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="seller_code">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>100</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="inventory">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>140</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
